### PR TITLE
Create getComputerInfo.sh

### PR DESCRIPTION
**Pull request by SKeenan07**
This BitBar plugin gets the computer name, battery condition, and IP address.

### CODE REVIEW

File: System/getComputerInfo.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# <bitbar.title>Get Computer Info</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Sarah Keenan</bitbar.author>
+# <bitbar.author.github>SKeenan07</bitbar.author.github>
+# <bitbar.desc>This plugin gets the IP address and the computer name.</bitbar.desc>
+# <bitbar.image>https://github.com/SKeenan07/portfolio/blob/master/images/BitBarPlugin.png</bitbar.image>
```
Comment (matryer) [inline System/getComputerInfo.sh:8]: please use `https://raw.githubusercontent.com/SKeenan07/portfolio/master/images/BitBarPlugin.png?raw=true`

**Review by SKeenan07 — COMMENTED** [review]
Updated link

**Comment by matryer** [comment]
@SKeenan07 I wonder if you could update the `<bitbar.abc>` metadata to `<xbar.abc>`?

**Comment by LivelyCarpet87** [comment]
Hello @SKeenan07, you may want to chmod +x your script to make it executable. It appears to me that's why the checks failed.

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**